doc/ics/yaml/gipsprojekt_de.yaml

```diff
@@ -3,15 +3,26 @@ title: Gipsprojekt
 url: https://www.gipsprojekt.de/
 howto:
   en: |
-    - Go to the Abfuhrkalender url of your service provider (like <https://www.gipsprojekt.de/featureGips/Gips?Anwendung=Abfuhrkalender&Mandant=Heidelberg&Abfuhrkalender=Heidelberg>) and click on your location/street.  
-    - Right-click -> copy the url of the Im iCalendar-Format abonnieren/speichern.
-    - Replace the `url` in the example configuration with this link.
-    - Replace the Jahr argument with `{%Y}` like in the example configuration. This way the year will be automatically updated.
+    - Go to the collection schedule URL of your service provider (like <https://www.stadtwerke-speyer.de/muellkalender>) and click on your location/street.
+    - Right-click the "Im iCalendar-Format abonnieren/speichern" link on the bottom half of the page and copy the URL.
+    - Replace the `url` in the example configuration with the copied link.
```

Comment:
I don't like `Replace the `url` in the example configuration with the copied link.` anymore, as it now also appears in the GUI configuration where there is no example. I remove it every time I touch an ICS source containing this sentence.

I'd prefer something like: "Use this URL as the `url` parameter"